guides/common/modules/proc_upgrading-the-external-database-operating-system.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Upgrading_the_External_Database_Operating_System{context}"]
+= Upgrading the external database operating system
+
+You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
+ifdef::katello,orcharhino,satellite[]
+For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
+endif::[]
+
+.Procedure
+. Create a backup.
+. Restore the backup on the new server.
+. If {Project} reaches the new database server via the old name, no further changes are required.
+Otherwise reconfigure {Project} to use the new name:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-db-host _newpostgres.example.com_ \
+--katello-candlepin-db-host _newpostgres.example.com_ \
+--foreman-proxy-content-pulpcore-postgresql-host _newpostgres.example.com_
+----
```

Comment:
But actually, I have one question: Is there something a user is expected to do to make sure the server is reachable? In other words, how can the user tell if the server is reachable? It might be a good idea to add one extra step, something like:

```suggestion
. Verify whether {Project} can reach the new database:
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
<command>
----
. If {Project} can reach the new database server by using the old name, no further changes are required.
Otherwise, reconfigure {Project} to use the new name:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# {foreman-installer} \
--foreman-db-host _newpostgres.example.com_ \
--katello-candlepin-db-host _newpostgres.example.com_ \
--foreman-proxy-content-pulpcore-postgresql-host _newpostgres.example.com_
----
```

Reply:
The user here should know if their new server has the same old hostname or not -- it's dependent on how they decide to set up the new system.
I think it doesn't hurt to add a test step, though. I'll have to find a command, I think we already have one documented somewhere.

Reply:
```
# PGPASSWORD='_Foreman_database_password' psql -h _postgres.example.com_  -p 5432 -U foreman -d foreman -c "SELECT 1 as ping"
```